### PR TITLE
Initial commit of debarred-entity schema

### PR DESCRIPTION
**Pull request by CountCulture**

### CODE REVIEW

File: schemas/includes/sanction.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        "legal_basis": {
+          "type": ["array","null"],
+          "items": {
+            "$ref": "legislation.json"
+          }
+        },
+        "sanction_jurisdictions": {
+          "description": "list of jurisdictions that this sanction applies to",
+          "type": "array",
+          "items": "String"
```
Comment (inglesp) [inline schemas/includes/sanction.json]: Shouldn't this be `"items": {"type": "string"}`?

File: schemas/includes/legislation.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "format": "non-blank",
+          "description": "name of the the legislation"
+        },
+        "url": {
+          "type": "string",
+          "format": "non-blank"
```
Comment (skenaja) [inline schemas/includes/legislation.json]: Not all laws are published online

Comment (CountCulture) [inline schemas/includes/legislation.json]: It's not required, just can't be blank if present

File: schemas/includes/sanction.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      "description": "A sanction issued by a government or regulatory body to an entity to prevent it operating (e.g. disqualified director) or ban others from dealing with it (e.g. a OFAC sanctioned entity).",
+      "type": "object",
+      "properties": {
+        "sanction_type": {
+          "enum": [
+            "disqualification",
+            "sanctions_listing",
+            "debarrment"
+          ],
+          "description": "types of sanction - one of 'disqualification', 'sanctions_list'"
```
Comment (skenaja) [inline schemas/includes/sanction.json]: add debarrment to description?

File: spec/sample-data/valid/sanctioned-entity-01.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      "url": "https://www.gov.uk/search-the-register-of-disqualified-company-directors",
+      "publisher": {
+        "name": "Companies House UK",
+        "jurisdiction": "gb",
+        "website": "https://www.gov.uk/government/organisations/companies-house"
+      }
+    },
+    "sanctions": [
+      {
+        "sanction_issuer": "UNDERTAKING",
```
Comment (skenaja) [inline spec/sample-data/valid/sanctioned-entity-01.json]: Isn't this meant to be an Organisation?

File: spec/sample-data/valid/sanctioned-entity-02.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        "website": "https://www.gov.uk/government/organisations/office-of-financial-sanctions-implementation"
+      }
+    },
+    "sanctions": [
+      {
+        "sanction_type": "sanctions_listing",
+        "sanction_jurisdictions": ["gb"],
+        "description": "Commander of the Western Military Region for the Sudanese Armed Forces, Sudan",
+        "notes": "Remains listed as Gaffar Mohamed Elhassan by the EU",
+        "cause": "UK Treasury Financial Sanctions Target"
```
Comment (skenaja) [inline spec/sample-data/valid/sanctioned-entity-02.json]: Cause is missing from schema definition

File: spec/sample-data/valid/sanctioned-entity-01.json (hunk 4 above)
Comment (CountCulture) [inline spec/sample-data/valid/sanctioned-entity-01.json]: or string

File: schemas/includes/sanction.json [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+          }
+        },
+        "sanction_jurisdictions": {
+          "description": "list of jurisdictions that this sanction applies to",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "sanction_issuer": {
```
Comment (skenaja) [inline schemas/includes/sanction.json]: Description would be helpful here

File: schemas/includes/sanction.json (hunk 3 above)
Comment (CountCulture) [inline schemas/includes/sanction.json]: and correct spelling: debarment

**Comment by CountCulture** [comment]
@skenaja I think I've addressed all these now

**Pull request merged**